subgraphs/_reference_/src/sdk/protocols/generic/index.ts

```diff
@@ -0,0 +1,147 @@
+import { Address, Bytes, BigInt, ethereum } from "@graphprotocol/graph-ts";
+
+import { ProtocolManager } from "./protocol";
+import { EventsManager } from "./event";
+import { PoolsManager } from "./pool";
+import { AccountsManager } from "./account";
+import { TokensManager, TokenInitializer } from "./token";
+import { TokenPricer } from "../config";
+import { Config } from "./config";
+import { BIGINT_ZERO, ZERO_ADDRESS } from "../../util/constants";
+
+export class CustomEventType {
```

Comment:
Make this type common for all the SDKs so we don’t re-implement it every time. You can then remove it from bridges too

Reply:
Added to utils!